### PR TITLE
docs: add Ferdinand and Johannes to GDE resources

### PR DESCRIPTION
**Pull request by fmalcher**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [x] Other: AIO marketing content

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

// CC @johanneshoppe

### CODE REVIEW

**Comment by JohannesHoppe** [comment]
@googlebot I consent.

**Review by kapunahelewong — CHANGES_REQUESTED** [review]
Thanks for these, @fmalcher! Just a couple of suggestions.

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -904,5 +904,21 @@
...
+    "website": "https://angular.schule",
+    "bio": "Ferdinand is a developer, trainer and international speaker from Leipzig, Germany. He is one author of the German Angular book and loves to share knowledge: That's why he writes books and blog posts, speaks at meetups and conferences, does Angular training and works as a consultant in his company \"Angular.Schule\".",
+    "groups": ["GDE"]
+  },
+  "johanneshoppe": {
+    "name": "Johannes Hoppe",
+    "picture": "johanneshoppe.jpg",
+    "twitter": "johanneshoppe",
+    "website": "https://angular.schule",
+    "bio": "Johannes works as a software architect and consultant for Angular. He is one author of the German Angular book and a passionate speaker. With his company \"Angular.Schule\" he offers public Angular training and consultancy. Johannes is head of the Angular Meetup in Heidelberg, Germany, and is engaged in a couple of open source projects.",
```
Comment (kapunahelewong) [inline aio/content/marketing/contributors.json]:
```suggestion
    "bio": "Johannes works as a software architect and consultant for Angular. He is co-author of the German Angular book and a passionate speaker. With his company \"Angular.Schule\" he offers public Angular training and consultancy. Johannes is head of the Angular Meetup in Heidelberg, Germany, and is engaged in a couple of open source projects.",
```

**Comment by fmalcher** [comment]
Thanks @kapunahelewong! 🙂 fixed

**Review by kapunahelewong — APPROVED** [review]
LGTM. Thank you, @fmalcher 🌟

**Comment by JohannesHoppe** [comment]
@kapunahelewong It would be really great if this PR could be merged! 😉 👍

**Comment by fmalcher** [comment]
Rebased against `master` so that it is mergeable. Would you mind merging? 😎 @StephenFluin Thanks!

**Comment by JohannesHoppe** [comment]
waiting...

**Review by StephenFluin — APPROVED** [review]

**Comment by fmalcher** [comment]
Rebased against current `master` to make it mergeable 👍

**Pull request closed without merging**